bin/check

```diff
@@ -4,7 +4,7 @@ command -v docker > /dev/null 2>&1 || {
   exit 1
 }
 
-docker version | grep -Fq "Server version" || {
+docker version | egrep -q "(Server version|Server:)" || {
```

Comment:
Since this check was never meant to be super strict anyway, what do you think about `grep -Fq "Server"`?

If you'd like to keep the or-pattern, I would just escape the pipe since it's only the one character (and the parens are redundant): `grep -q "Server version\|Server:"`.

If you'd like to keep the extended option, please switch the deprecated `egrep` out for `grep -E`.